build/cli/src/lib.rs

```diff
@@ -667,25 +582,10 @@ impl Processor {
         .boxed()
     }
 
-    pub fn build_old_ide(
-        &self,
-        params: arg::ide::BuildInput<Gui>,
-    ) -> BoxFuture<'static, Result<ide::Artifact>> {
-        let arg::ide::BuildInput { gui, project_manager, output_path, electron_target } = params;
-        let input = ide::BuildInput {
-            gui: self.get(gui),
-            project_manager: self.get(project_manager),
-            version: self.triple.versions.version.clone(),
-            electron_target,
-            artifact_name: "ide".into(),
-        };
-        self.build_ide(input, output_path)
-    }
-
     pub fn build_new_ide(
         &self,
-        params: arg::ide2::BuildInput,
-    ) -> BoxFuture<'static, Result<ide2::Artifact>> {
+        params: arg::ide::BuildInput,
+    ) -> BoxFuture<'static, Result<ide::Artifact>> {
```

Comment:
overlooked "new_ide"?

Reply:
> overlooked "new_ide"?

Well, the `_ide` function already exists but it was extracted only due to new/old ide reuse. I'll inline it back.